app/src/app/logToFile.js

```diff
@@ -0,0 +1,24 @@
+import {app} from 'electron'
+import fs from 'fs'
+import {normalize} from 'path'
+
+import {isWindows} from './utils'
+
+/**
+ * Escapes a path so that it can be printed to console on windows and mac in
+ * a usable form to copy paste it in the shell.
+ */
+function escapePath(path) {
+  const escapeChar = isWindows() ? '^' : '\\'
+  return path.replace(/( |\(|\))/g, `${escapeChar}$1`)
```

Comment:
dunno, needs to be tested, I got that one from stackoverflow, didn't want to invest time to fully understand and test it because its just for one relatively simple use case and it seems to work.